### DISCUSSION TITLE
Custom candlestick colors and more "styles".

### DISCUSSION THREAD

**Discussion by bukowa** [grafana/grafana#66632] [Dashboards, Panels & Visualizations]
1. Hey i cannot find a way to colorize candlesticks based on other data in a given row, example:

````
time,open,color
...,1,08,red

````
Or setting candle color based on condition with an option to override colors based on condition in the panel edit view.
(if -1 = color red)
````
time,open,condition
...,1,08,-1
````

2. I cannot find a way to plot different icons other than "lines/bars/points".

3. Imagine this data, when; in order to plot a point on the given candlestick i have to transfer the condition "1" into the "open" price in order to show points on the candlesticks:
````
time,open,point
...,1.25,1
````
````
time,open,point
...,1.25,1.25
````
![image](https://user-images.githubusercontent.com/38087302/232218494-bb44ca1c-4543-4493-beb0-5cb734dec94a.png)

I am missing something to make candlestick chart more "plottable" ? I am not advanced grafana users and some guidance and discussion would be great. Grafana candlesticks are awesome but from my understanding missing some important features which i do not know how to plot.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82738. Feel free to continue the discussion around this there. Thank you!
